### PR TITLE
AT_16.02_001| Rename Multibranch Pipeline

### PR DESCRIPTION
**Pull request by vlada4nik**
https://trello.com/c/gd9IUGus/879-at1602001-multibranch-pipeline-rename-multibranch-pipeline

### CODE REVIEW

**Comment by S-Lanchik** [comment]
Add PR link to AT

**Comment by vlada4nik** [comment]
> Add PR link to AT

added, thanks for comment.

File: cypress/e2e/multibranchPipelineRenameNew.cy.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+/// <reference types="cypress"/>
+describe('Multibranch Pipeline Rename', ()=>{
+    let name='mew name'
+    let name2='QA_V'
```
Comment (Rom-T) [inline cypress/e2e/multibranchPipelineRenameNew.cy.js:4]: Please move data in the fixtures

File: cypress/e2e/multibranchPipelineRenameNew.cy.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    beforeEach('Precondition: Create Multibranch Pipeline',()=>{
+        cy.get('a[href="newJob"]').click()
+        cy.get('li[tabindex="0"]').contains('Multibranch Pipeline').click()
+        cy.get('input#name').type(name)
+        cy.get('div[class="btn-decorator"]').click()   
+    })
+    it('Multibranch Pipeline Rename-inside the selected Multibranch Pipeline',()=>{
+        cy.get('[class="model-link"]').contains('Dashboard').click()
+        cy.get('a[href="job/mew%20name/"]').click()
+        cy.get('a[href="/job/mew%20name/confirm-rename"]').click()
```
Comment (Rom-T) [inline cypress/e2e/multibranchPipelineRenameNew.cy.js:14]: Try to find a locator wherever the name Multibranch Pipeline is included

File: cypress/e2e/multibranchPipelineRenameNew.cy.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+        cy.get('input[checkdependson="newName"]').clear()
+        cy.get('input[checkdependson="newName"]').type(name2)
```
Comment (Rom-T) [inline cypress/e2e/multibranchPipelineRenameNew.cy.js:16]: Actions for 1 element can be listed through a dot
cy.get('input[checkdependson="newName"]').clear().type(name2)

File: cypress/e2e/multibranchPipelineRenameNew.cy.js [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+        cy.get('button[name="Submit"]').click()
+
+        cy.get('div[id="main-panel"]').should('contain', 'QA_V')
+        cy.get('div[id="page-body"]').should('contain', 'QA_V')
```
Comment (Rom-T) [inline cypress/e2e/multibranchPipelineRenameNew.cy.js:20]: Please move `'QA_V'` in the fixtures

File: cypress/e2e/multibranchPipelineRenameNew.cy.js (hunk 1 above)
Comment (vlada4nik) [reply cypress/e2e/multibranchPipelineRenameNew.cy.js]: Done. Thanks

File: cypress/e2e/multibranchPipelineRenameNew.cy.js (hunk 4 above)
Comment (vlada4nik) [reply cypress/e2e/multibranchPipelineRenameNew.cy.js]: Done.Thanks

File: cypress/e2e/multibranchPipelineRenameNew.cy.js (hunk 3 above)
Comment (vlada4nik) [reply cypress/e2e/multibranchPipelineRenameNew.cy.js]: Done.Thanks

File: cypress/e2e/multibranchPipelineRenameNew.cy.js (hunk 2 above)
Comment (vlada4nik) [reply cypress/e2e/multibranchPipelineRenameNew.cy.js:14]: Sorry I don't really get what exactly should I do. I don't know how to find you in SLACK, may you please contact me @Vlada Nikitchenko? Or you can explain it here. Thank you!

**Review by Rom-T — APPROVED** [review]
LGTM

**Review by IraKalinka — APPROVED** [review]

**Pull request merged**